.changeset/beige-items-teach.md

```diff
@@ -0,0 +1,11 @@
+---
+'@spreadshirt/backstage-plugin-s3-viewer-backend': minor
+---
+
+Add new cluster location using IAM roles. Defined by using the platform
+type `iam-role`.
+
+This locator is similar to the `config` and `radosgw-admin`, but 
+it doesn't need any credentials to be set. Check the 
+[README file](../plugins/s3-viewer-backend/README.md#configuration) 
```

Comment:
This long probably won't work, as it needs to work from the releases page I think?  A full absolute link would probably be better here.

Reply:
Changed the link